### PR TITLE
User login and sign up

### PR DESCRIPTION
**Pull request by bakijulia**

### CODE REVIEW

File: jade_reader/jade_reader/UserSignUpWindow.cpp [hunk 1]
```diff
@@ -19,13 +22,23 @@ UserSignUpWindow::UserSignUpWindow(QWidget *parent): QDialog(parent) {
...
   mainLayout->addWidget(backButton);
 
-  connect(backButton, &QAbstractButton::clicked, this, &UserSignUpWindow::onPushButtonEvent);
+  connect(backButton, &QAbstractButton::clicked, this, &UserSignUpWindow::onBackButtonEvent);
+  connect(signUpButton, &QAbstractButton::clicked, this, &UserSignUpWindow::onSignUpButtonEvent);
 }
 
-void UserSignUpWindow::onPushButtonEvent() {
+void UserSignUpWindow::onBackButtonEvent() {
   emit sendRequest();
```
Comment (bmakszim) [inline jade_reader/jade_reader/UserSignUpWindow.cpp]: emit macro is not necessary here

File: jade_reader/jade_reader/UserLoginWindow.cpp [hunk 2]
```diff
@@ -1,25 +1,33 @@
...
   mainLayout->addWidget(signUpButton);
-  connect(signUpButton, &QAbstractButton::clicked, this, &UserLoginWindow::onEvent);
+
+  connect(signUpButton, &QAbstractButton::clicked, this, &UserLoginWindow::onSignUpButtonEvent);
+  connect(signInButton, &QAbstractButton::clicked, this, &UserLoginWindow::onSignInButtonEvent);
 }
 
-void UserLoginWindow::onEvent() {
+void UserLoginWindow::onSignUpButtonEvent() {
   emit sendRequest();
```
Comment (bmakszim) [inline jade_reader/jade_reader/UserLoginWindow.cpp]: It works without the emit macro as well.

File: jade_reader/jade_reader/WindowManager.cpp [hunk 3]
```diff
@@ -7,6 +7,7 @@
 WindowManager::WindowManager() {
   firstPageWidget = new UserLoginWindow;
   secondPageWidget = new UserSignUpWindow;
```
Comment (bmakszim) [inline jade_reader/jade_reader/WindowManager.cpp]: LoginPageWidget and SignUpPageWidget or something similar might be better names for the QWidget variables.

File: jade_reader/jade_reader/UserSignUpWindow.cpp (hunk 1 above)
Comment (bakijulia) [reply jade_reader/jade_reader/UserSignUpWindow.cpp]: done

File: jade_reader/jade_reader/UserLoginWindow.cpp (hunk 2 above)
Comment (bakijulia) [reply jade_reader/jade_reader/UserLoginWindow.cpp]: done :)

File: jade_reader/jade_reader/WindowManager.cpp (hunk 3 above)
Comment (bakijulia) [reply jade_reader/jade_reader/WindowManager.cpp]: fixed

**Pull request closed without merging**